packages/instantsearch.js/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
what tool is this one needed for?

Reply:
Just forgot to remove this while I was experimenting haha